runtime-sdk/modules/evm/src/precompile/mod.rs

```diff
@@ -125,6 +127,8 @@ impl<Cfg: Config, B: EVMBackendExt> PrecompileSet for Precompiles<'_, Cfg, B> {
             (1, 0, 6) => confidential::call_sign(handle),
             (1, 0, 7) => confidential::call_verify(handle),
             (1, 0, 8) => confidential::call_curve25519_compute_public(handle),
+            (1, 0, 9) => gas::call_gas_used(handle),
+            (1, 0, 10) => gas::call_pad_gas(handle),
```

Comment:
Can you also update the `fuzz/precompile.rs` to include these addresses?

Reply:
updated